apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/FeatureFlagsSettings/GroupedFeatures.tsx

```diff
@@ -1,5 +1,5 @@
 import { groupedFeatures } from "@ledgerhq/live-common/featureFlags/groupedFeatures";
-import { useFeatureFlags } from "@ledgerhq/live-common/featureFlags/provider";
+import { useFeatureFlags } from "@ledgerhq/live-common/featureFlags/FeatureFlagsContext";
```

Comment:
It does not matter, but everything in `featureFlags` that can be exported is exported from `index.ts` no ? You can directly import from `@ledgerhq/live-common/featureFlags/index`

Reply:
ah yep indeed, here it was auto renaming in action